src/main/flight/pid.c

```diff
@@ -86,13 +90,30 @@ void pidResetErrorAngle(void)
 
 void pidResetErrorGyro(void)
 {
-    errorGyroI[ROLL] = 0;
-    errorGyroI[PITCH] = 0;
-    errorGyroI[YAW] = 0;
+    int axis;
+    for (axis = 0; axis < 3; axis++) {
+        errorGyroI[axis] = 0;
+        errorGyroIf[axis] = 0.0f;
+    }
+}
+
+float scaleItermToRcInput(int axis) {
```

Comment:
BTW: iTermScaler will prevent I-TERM action until it is close to 1 (I term is integrated, this factor works as 'leak'). With 4ms looptime, this will result in 4s delay with zero I-term.